src/Api.php

```diff
@@ -0,0 +1,117 @@
+<?php declare(strict_types=1);
+
+namespace Bannerbear;
+
+
+use function curl_close;
+use function curl_errno;
+use function curl_error;
+use function curl_exec;
+use function curl_getinfo;
+use function curl_init;
+use function curl_setopt;
+use function curl_setopt_array;
+use function json_decode;
+use function json_encode;
```

Comment:
Not sure about importing these functions from the root namespace - the original file does not import:

https://github.com/yongfook/bannerbear-php/blob/493b93685c6f2bb1130c0271f27389e204a430a9/src/BannerbearClient.php#L1-L6

Reply:
It's just micro optimizations

Reply:
I know, but it's an unrelated change.

We could use [`friendsofphp/php-cs-fixer`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer) with the [`global_namespace_import`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.23.0/doc/rules/import/global_namespace_import.rst) fixer configured and enabled to make that change in a separate pull request.

What do you think?

Reply:
I reverted the imports